packages/cli/src/services/orchestration/webhook/handleCommandMessageWebhook.ts

```diff
@@ -1,6 +1,81 @@
-import { handleCommandMessageMain } from '../main/handleCommandMessageMain';
+import { ExternalSecretsManager } from '@/ExternalSecrets/ExternalSecretsManager.ee';
+import { License } from '@/License';
+import { MessageEventBus } from '@/eventbus';
+import Container from 'typedi';
+import { Logger } from 'winston';
+import { messageToRedisServiceCommandObject, debounceMessageReceiver } from '../helpers';
+import config from '@/config';
 
 export async function handleCommandMessageWebhook(messageString: string) {
-	// currently webhooks handle commands the same way as the main instance
-	return await handleCommandMessageMain(messageString);
+	const queueModeId = config.getEnv('redis.queueModeId');
+	const isMainInstance = config.getEnv('generic.instanceType') === 'main';
+	const message = messageToRedisServiceCommandObject(messageString);
+	const logger = Container.get(Logger);
+
+	if (message) {
+		logger.debug(
+			`RedisCommandHandler(main): Received command message ${message.command} from ${message.senderId}`,
+		);
+
+		const selfSendingAllowed = [
+			'add-webhooks-triggers-and-pollers',
+			'remove-triggers-and-pollers',
+		].includes(message.command);
```

Comment:
This won't be necessary since webhooks never add or remove triggers and pollers (no internal API)

Reply:
Good catch, addressed!